pkg/manifests/manifests.go

```diff
@@ -2147,13 +2148,9 @@ func (f *Factory) MetricsServerDeployment(apiAuthSecretName string, kubeletCABun
 	return dep, nil
 }
 
-func (f *Factory) MetricsServerSecret(tlsSecret *v1.Secret, apiAuthConfigmap *v1.ConfigMap) (*v1.Secret, error) {
+func (f *Factory) MetricsServerClientCASecret(apiAuthConfigmap *v1.ConfigMap) (*v1.Secret, error) {
 	data := make(map[string]string)
 
-	for k, v := range tlsSecret.Data {
```

Comment:
This is followup change as in https://github.com/openshift/cluster-monitoring-operator/pull/2315